AppCenter/AppCenter/include/AppCenter.h

```diff
@@ -0,0 +1 @@
+../AppCenter.h
```

Comment:
Is it better to move public headers to `./include/`? Does it work?

Reply:
Otherwise we have to keep in mind whenever we add a new public header, we have to add this in `include`.

Reply:
> Does it work?

No, it doesn't work well. We store headers in not one folder and reuse some Core headers in other modules. Symlinks simplifies things here.

> Otherwise we have to keep in mind whenever we add a new public header, we have to add this in include as well.

Yes, but we prepared integration tests on CI that should detect this kind of issues.